auto_dev/cli_executor.py

```diff
@@ -27,8 +27,25 @@ def __init__(self, command: Union[str, List[str]], cwd: Optional[str] = None):
         self.return_code = None
         self.exception = None
 
-    def execute(self, stream=False, verbose: bool = True, shell: bool = False, env_vars: Optional[dict] = None) -> bool:
-        """Execute the command."""
+    def execute(
+        self,
+        stream=False,
+        verbose: bool = True,
+        shell: bool = False,
+        env_vars: Optional[dict] = None,
+        interactive: bool = False,
+    ) -> bool:
+        """Execute the command.
+
+        Args:
+            stream: Whether to stream output
+            verbose: Whether to show verbose output
+            shell: Whether to run in shell
+            env_vars: Environment variables to set
+            interactive: Whether to allow interactive input/output
+        """
+        if interactive:
```

Comment:
i assume as we cannot have have both interactive and stream we should verify in the init function